extensions/objects/templates/FilepathEditor.js

```diff
@@ -590,7 +591,11 @@ oppia.directive('filepathEditor', [
           };
           if (updateParent) {
             AlertsService.clearWarnings();
-            $scope.$parent.value = filename;
+            $scope.$parent.value = {
```

Comment:
@apb7 Hi, we want to get rid of this `$scope.$parent` in */templates*?

Reply:
Hi @vojtechjelinek, yes! We should only use the scope object to access the parent's properties.

Reply:
Hi @vojtechjelinek , @apb7 Can you please help me with it. Thanks!

Reply:
Hi @ishucr7, yes please update your branch and then, since `value` is being passed from the parent to the directive via the scope object using `=` prefix, you can change `$scope.$parent.value` to `$scope.value`. This should work out. Also, see the other directives once. Thanks!

Reply:
Okay, Thanks @apb7 .

Reply:
Done!